src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -5,8 +5,8 @@
 import java.util.HashMap;
 
 public class LottoNumber {
-    private int number;
-    private static HashMap<Integer, LottoNumber> lottoNumbers = new HashMap<>();
+    private final int number;
+    private static final HashMap<Integer, LottoNumber> lottoNumbers = new HashMap<>();
```

Comment:
```suggestion
    private static final HashMap<Integer, LottoNumber> lottoNumbers = new HashMap<>();
     
    private final int number;
```

상수를 먼저 쓰고, 인스턴스 변수를 쓰면 어떨까요?